pkg/frontend/backend/jaeger-storage/backend.go

```diff
@@ -165,69 +168,113 @@ func (backend *Backend) List(
 		filterTags["cluster"] = params.OperationName
 	}
 
-	newParams := &spanstore.TraceQueryParameters{
-		ServiceName:  "object",
-		Tags:         filterTags,
-		StartTimeMin: params.StartTimeMin,
-		StartTimeMax: params.StartTimeMax,
-		DurationMin:  params.DurationMin,
-		DurationMax:  params.DurationMax,
-		NumTraces:    params.NumTraces,
-	}
+	// TODO support additional user-defined trace sources
+	var traces []*model.Trace
+	for _, traceSource := range zconstants.KnownTraceSources(false) {
```

Comment:
Object spans always start at :00 or :30, while audit/event spans start at the exact event timestamp.

If the user searches for events between :16 and :20, we want to search traces containing spans within this range. If we just search objects, we will get a lot of traces that actually have no spans in this range, and actually relevant traces will not display in the list since there is usually a limit.